server.js

```diff
@@ -134,12 +134,14 @@ io.on("connection", function(socket) {
 	// client sent update properties for task
 	socket.on("update", function(data) {
 		// required
-		if (!socket.room || !data.id) return;
+		console.log(data)
+		//if (!socket.room || !data.id) return;
```

Comment:
this should be left in to prevent db errors. if this line is blocking updates from happening, then we need to figure out why the room or id is null.